src/main/java/org/jenkinsci/test/acceptance/plugins/job_dsl/JobDSLBuildStepAdvanced.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkinsci.test.acceptance.plugins.job_dsl;
+
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.PageObject;
+
+import java.net.URL;
+
+/**
+ * Encapsulates the PageObject of the advanced area of the Job DSL plugin.
+ *
+ * @author Maximilian Oeckler
+ */
+public class JobDSLBuildStepAdvanced extends PageObject {
```

Comment:
Wouldn't it make more sense to include the code of this class into JobDSLBuildStep? 

Then all setters for advanced options would have a call to advanced.click(). See https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/java/org/jenkinsci/test/acceptance/plugins/analysis_core/AnalysisSettings.java#L105 for an example